shells/firefox/manifest.json

```diff
@@ -0,0 +1,55 @@
+{
+  "manifest_version": 2,
+  "name": "React Developer Tools",
+  "description": "Adds React debugging tools to the Chrome Developer Tools.",
+  "version": "2.3.1",
+
+  "applications": {
+    "gecko": {
+      "id": "extension@react.devtools",
+      "strict_min_version": "54.0"
```

Comment:
(Belated follow-up...) My own testing seemed to suggest that everything _except_ the theme worked in 54, and that degrades gracefully to the light theme.